cmd/check-gh-automation/README.md

```diff
@@ -12,6 +12,7 @@ check-gh-automation \
 --config-path=/release/core-services/prow/02_config/_config.yaml \
 --supplemental-prow-config-dir=/release/core-services/prow/02_config \
 --job-config-path=/release/ci-operator/jobs/ \
+--plugin-config=/release/core-services/prow/02_config/ViaQ/_pluginconfig.yaml \
```

Comment:
I think we will end up using the `supplemental-plugin-config-dir` option in the real check, but this is okay for now. We will likely update this doc once the PR to add the options is in.